minos/networks/rest/builders.py

```diff
@@ -25,7 +26,8 @@ class RestBuilder:
 
     __slots__ = "_config", "_app"
 
-    def __init__(self, config: MinosConfig, app: web.Application = web.Application()):
+    def __init__(self, config: MinosConfig, app: web.Application = web.Application(), **kwargs):
```

Comment:
Instead of directly passing the `config` instance to the constructor of the class, could you implement the `_from_config` method and only pass the `config.rest.endpoints` to the constructor?

Reply:
Done!